pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -29,19 +31,26 @@ import (
 
 const (
 	binaryPath     = "/usr/bin/virt-chroot"
```

Comment:
```
ERROR: git tree state is not clean!
You probably need to run 'make generate' or 'make' and commit the changes
On branch main
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/virt-handler/virt-chroot/virt-chroot.go
no changes added to commit (use "git add" and/or "git commit -a")
diff --git a/pkg/virt-handler/virt-chroot/virt-chroot.go b/pkg/virt-handler/virt-chroot/virt-chroot.go
index d022eb87d..0bc021af7 100644
--- a/pkg/virt-handler/virt-chroot/virt-chroot.go
+++ b/pkg/virt-handler/virt-chroot/virt-chroot.go
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	binaryPath     = "/usr/bin/virt-chroot"
+	binaryPath = "/usr/bin/virt-chroot"
 )
```

Reply:
thanks. patched. I'm blindly changing without compiling ;-)

Reply:
I think that was the last issue. Let's see if the CI passes.